### DISCUSSION TITLE
FF works correctly with Roland G-800

### DISCUSSION THREAD

**Discussion by frisc1** [keirf/flashfloppy#764] [Show and tell]
Hi,

Just to confirm that FF is working correctly with the Roland G-800 keyboard synthesizer. 

Had to add in FF.CFG only the line:
interface = jppc-hdout

Jumper only on S0.

Maybe this info can be added to the Roland host page:
 https://github.com/keirf/flashfloppy/wiki/Host-Platforms#roland

Regards,
Frisc1

**Comment by keirf** [comment]
Done, thanks for the report!

**Reply by gxcv22** [reply]
I can only use Index mode on G-800.  Is it possible to make Normal mode works?
